src/components/comments/comments.html

```diff
@@ -0,0 +1,5 @@
+<div class="comments">
+  <comments-header adding-comment="$ctrl.isAddingComment" on-start-adding-comment="$ctrl.startAddingComment();" on-cancel-adding-comment="$ctrl.cancelAddingComment();"></comments-header>
```

Comment:
remove the trailing semicolon at the end of the expression, is useless and it seems dirty